docs/reference/slm/getting-started-slm.asciidoc

```diff
@@ -143,44 +148,33 @@ next time the policy will be executed.
-    } ,
-    "next_execution": "2019-04-24T01:30:00.000Z", <4>
-    "next_execution_millis": 1556048160000
+    "next_execution": "2019-04-24T01:30:00.000Z", <3>
+    "next_execution_millis": 1556048160000 
   }
 }
 --------------------------------------------------
 // TESTRESPONSE[skip:the presence of last_failure and last_success is asynchronous and will be present for users, but is untestable]
 
-<1> information about the last time the policy successfully initated a snapshot
-<2> the name of the snapshot that was successfully initiated
-<3> information about the last time the policy failed to initiate a snapshot
-<4> the next time the policy will execute
+<1> The name of the last snapshot that was succesfully initiated by the policy
+<2> When the snapshot was initiated
+<3> When the policy will initiate the next snapshot
 
-NOTE: This metadata only indicates whether the request to initiate the snapshot was
-made successfully or not - after the snapshot has been successfully started, it
-is possible for the snapshot to fail if, for example, the connection to a remote
+The response shows if the policy succeeded in _initiating_ a snapshot.
+However, that does not guarantee that the snapshot completed successfully. 
+It is possible for the initiated snapshot to fail if, for example, the connection to a remote
 repository is lost while copying files.
 
-If you're following along, the returned SLM policy shouldn't have a `last_failure`
-field - it's included above only as an example. You should, however, see a 
-`last_success` field and a snapshot name. If you do, you've successfully taken
-your first snapshot using SLM!
-
-While only the most recent sucess and failure are available through the Get Policy 
-API, all policy executions are recorded to a history index, which may be queried
-by searching the index pattern `.slm-history*`.
+Only the most recent success and failure are returned, 
+but all policy executions are recorded in the `.slm-history*` index.
```

Comment:
It's strange to call `.slm-history*` a single index, since it's a collection of indices behind an alias named `.slm-history`, we could either change this to "indices" or only mention the alias?